Backend/models/meeting.js

```diff
@@ -0,0 +1,35 @@
+import { DataTypes } from 'sequelize';
+import { sequelize } from '../database.js';
+
+export const Meeting = sequelize.define('Meeting', {
+  id: {
```

Comment:
do we need to include the state field to indicate if it's accepted? can add later as well

Reply:
Great suggestion. I had just set up a temporary table for structure purposes. I plan on going in more detail with the calendar as that's more what it is adjacent to.